src/content/docs/en/reference/api-reference.mdx

```diff
@@ -1699,9 +1699,13 @@ import { Code } from 'astro:components';
   <Code code={`const foo = 'bar';`} lang="js" inline />
   will be rendered inline.
 </p>
+<!-- Optional: defaultColor -->
+<Code code={`const foo = 'bar';`} lang="js" defaultColor={false} />
 ```
 
-This component provides syntax highlighting for code blocks at build time (no client-side JavaScript included). The component is powered internally by Shiki and it supports all popular [themes](https://shiki.style/themes) and [languages](https://shiki.style/languages). Plus, you can add your custom themes, languages, and [transformers](#transformers) by passing them to the `theme`, `lang`, and `transformers` attributes respectively.
+This component provides syntax highlighting for code blocks at build time (no client-side JavaScript included). The component is powered internally by Shiki and it supports all popular [themes](https://shiki.style/themes) and [languages](https://shiki.style/languages). Plus, you can add your custom themes, languages, [transformers](#transformers), and [default color theme](https://shiki.style/guide/dual-themes#without-default-color) by passing them to the `theme`, `lang`, `transformers`, and `defaultColor` attributes respectively.
+
+By default, this component will use the settings from your [Shiki configuration](/en/guides/markdown-content/#shiki-configuration).
```

Comment:
I just tested and it seems that the `<Code />` component does not inherit any configuration 😕

It think it is worth to point out that to avoid confusion.

Reply:
Interesting! So it doesn't even use the theme you've set or anything? I would have assumed that it would! (It's a better story to tell if it does!) I wonder how difficult that would be for us to add.... 

But yes, if it's not true, we won't say it! Thank you for testing!

Reply:
No problem!

I think it would not be so hard to add. One workaround would be to import astro config and then pass the specific part as props to the `Code` component. Something like this:

On the config file, export the shiki config:
```typescript
export const shikiConfig = {
  //...
};

// All the regular astro config...
```

Then on an astro file import the config and use it to pass props to the code component:
```astro
---
import { shikiConfig } from '../astro.config.ts';
---

<Code {...shikiConfig} />
```